src/native_main.nim

```diff
@@ -251,22 +251,21 @@ proc handleMessage(msg: MessageRecv): string =
 
     return $ %* reply # $ converts to string, %* converts to JSON
 
-let params = commandLineParams()
-if len(params) > 0 and params[0] == "restart":
-    when defined windows:
-        # need to check this twice so we fail properly on wrong usage of
-        # "restart" argument
-        if len(params) == 4:
-            restartWindowsStep2(firefoxPid = params[1].parseInt(),
-                profiledir = params[2], browsername = params[3])
-            quit(QuitSuccess)
-    quit(QuitFailure)
-else:
-    while true:
-        let strm = newFileStream(stdin)
-        let message = handleMessage(getMessage(strm))
-        let l = pack("@I", message.len)
-
-        write(stdout, l)
-        write(stdout, message) # %* converts the object to JSON
-        flushFile(stdout)
+when defined windows:
+    let params = commandLineParams()
+    # Usage: native_main.exe restart <Firefox PID> <profile dir> <browser exe name>
```

Comment:
Is it actually called `native_main.exe`? I thought I went to some lengths to call it just `native_main`

Reply:
Yes. That's what `nimble build` produces and what the installer installs it to. I think we discussed this already, but there's no need to call it anything else, since the installer sets the name of the binary in `tridactyl.json` anyway.

Reply:
Ah, I see. We drop the `.exe` when attaching the binary to the release via CI and then add the `.exe` back when we download it via the installer.